packages/wdio-browserstack-service/src/service.ts

```diff
@@ -110,6 +111,17 @@ export default class BrowserstackService implements Services.ServiceInstance {
         // added to maintain backward compatibility with webdriverIO v5
         this._browser = browser ? browser : globalThis.browser
 
+        // Healing Support:
+        if (!isBrowserstackSession(this._browser)) {
+            try {
+                await AiHandler.selfHeal(this._options, caps, this._browser)
+            } catch (err) {
+                if (this._options.selfHeal === true) {
+                    BStackLogger.debug(`Error while setting up self-healing: ${err}. Disabling healing for this session.`)
+                }
+            }
+        }
+
```

Comment:
Please move this try catch to self heal function itself, that will help reduce the clutter in this function

Reply:
This was requested in the last review